lma-browser-extension-stack/src/context/ProviderIntegrationContext.tsx

```diff
@@ -97,6 +97,8 @@ function IntegrationProvider({ children }: any) {
       setPlatform("Zoom");
     } else if (newMetadata && newMetadata.baseUrl && newMetadata.baseUrl === "https://app.chime.aws") {
       setPlatform("Amazon Chime");
+    } else if (newMetadata.baseUrl === "https://teams.microsoft.com" || newMetadata.baseUrl === "https://teams.live.com") {
```

Comment:
Does `teams.microsoft.com` work?  Everything else seems to be specific to `teams.live.com`  
Can you add clarifying comments to explain the difference and what works and what is yet to be done.. Thanks!

Reply:
Yes both `teams.microsoft.com` and `teams.live.com` work